jck/compiler.lang/playlist.xml

```diff
@@ -728,6 +728,29 @@
 			<group>jck</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>jck-compiler-lang-EXPR-group1_multijvm</testCaseName>
+		<disables>
+			<disable>
+				<comment>Disabled temporarily in multijvm mode</comment>
+				<impl>hotspot</impl>
+			</disable>
+		</disables>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(GEN_JTB) tests=lang/EXPR testsuite=COMPILER$(USEQ); \
+		$(START_MULTI_JVM_COMP_TEST); \
+		$(TEST_STATUS); \
+		$(GEN_SUMMARY) tests=lang/EXPR testsuite=COMPILER
+		</command>
+		<levels>
+			<level>dev</level>
+		</levels>
+		<groups>
+			<group>jck</group>
+		</groups>
+	</test>
```

Comment:
There is `jck-compiler-lang-EXPR-group1_multijvm` at https://github.com/adoptium/aqa-tests/blob/1305f48d1d779427b8c9b8a5050e93bed9cc7463/jck/compiler.lang/playlist.xml#L1144 
Why do we add `jck-compiler-lang-EXPR-group1_multijvm` here again and exclude it?

Reply:
I hadn't spotted that it was there (I don't usually modify this file) - fixed.